db/migrate/20230803022448_add_keyword_status_variant.rb

```diff
@@ -0,0 +1,9 @@
+class AddKeywordStatusVariant < ActiveRecord::Migration[7.0]
+  def change
+    execute <<-SQL
+      ALTER TYPE keyword_status ADD VALUE 'pending';
+    SQL
+
+    change_column_default :keywords, :status, 'pending'
```

Comment:
FYI @Goose97 as we haven't merge this migration so with this kind of new change we can also edit it directly in the original migration :D app/services/google/search_service.rb, but a new migration like this as also fine just FYI

Reply:
Thanks, good to know